sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2021-2022.bpmn

```diff
@@ -673,6 +655,126 @@
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=studentDataAdditionalTransportKm = 0</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="Flow_0f31yl5" sourceRef="Event_15d88qb" targetRef="Gateway_02qbvrn" />
+    <bpmn:sequenceFlow id="Flow_0qgamw3" sourceRef="Gateway_02qbvrn" targetRef="Gateway_0pzv80g" />
+    <bpmn:exclusiveGateway id="Gateway_1o496q2">
+      <bpmn:incoming>Flow_1k4pq3k</bpmn:incoming>
+      <bpmn:outgoing>Flow_0201yrq</bpmn:outgoing>
+      <bpmn:outgoing>Flow_04xv6y2</bpmn:outgoing>
+      <bpmn:outgoing>Flow_1ibgfq9</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:intermediateThrowEvent id="Event_15d88qb" name="Calculate Total Transportation Allowance">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=if offeringDelivered = &#34;onsite&#34;&#10;then&#10;  dmnPartTimeProgramYearMaximums.limitTransportationAllowance * offeringWeeks&#10;else&#10;  0" target="calculatedDataTotalTransportationAllowance" />
+        </zeebe:ioMapping>
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_0201yrq</bpmn:incoming>
+      <bpmn:outgoing>Flow_0f31yl5</bpmn:outgoing>
+    </bpmn:intermediateThrowEvent>
+    <bpmn:exclusiveGateway id="Gateway_02qbvrn">
+      <bpmn:incoming>Flow_0f31yl5</bpmn:incoming>
+      <bpmn:incoming>Flow_0ce608y</bpmn:incoming>
+      <bpmn:incoming>Flow_0sezwqd</bpmn:incoming>
+      <bpmn:outgoing>Flow_0qgamw3</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:intermediateThrowEvent id="Event_1lqekii" name="Calculate Total Transportation Allowance">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=min(&#10;  studentDataAdditionalTransportCost&#10;  ,&#10;  if (studentDataAdditionalTransportPlacement)&#10;  then&#10;    dmnPartTimeProgramYearMaximums.limitAdditionalTransportationAllowance - dmnPartTimeProgramYearMaximums.limitAdditionalTransportationPlacement&#10;  else&#10;    dmnPartTimeProgramYearMaximums.limitAdditionalTransportationAllowance&#10;)" target="calculatedDataAdditionalWeeklyTransportationCost" />
+          <zeebe:output source="=calculatedDataAdditionalWeeklyTransportationCost * min(offeringWeeks, studentDataAdditionalTransportWeeks) * dmnPartTimeProgramYearMaximums.limitAdditionalTransportationKMReduction" target="calculatedDataAdditionalTransportationMax" />
+          <zeebe:output source="=if all(offeringDelivered = &#34;onsite&#34;, offeringWeeks &#62;= studentDataAdditionalTransportWeeks)&#10;then&#10;  (offeringWeeks - studentDataAdditionalTransportWeeks) * dmnPartTimeProgramYearMaximums.limitTransportationAllowance + calculatedDataAdditionalTransportationMax&#10;else&#10;  min(calculatedDataAdditionalTransportationMax, studentDataAdditionalTransportCost * min(offeringWeeks, studentDataAdditionalTransportWeeks))" target="calculatedDataTotalTransportationAllowance" />
```

Comment:
Another question (more to the business) @CarlyCotton : Are the students going to get the transportation allowance when they are not enrolled `onsite` as in enrolled `online`?